at_commons/lib/src/verb/llookup_verb_builder.dart

```diff
@@ -15,18 +16,20 @@ import 'package:at_commons/src/verb/verb_util.dart';
 /// // Lookup a credit card number that is accessible only by Bob
 ///    var builder = LlookupVerbBuilder()..key=’@bob:credit_card’..atSign=’bob’;
 class LLookupVerbBuilder implements VerbBuilder {
-  /// the key of [atKey] to llookup. [atKey] can have either public, private or shared access.
-  String? atKey;
-
-  /// atSign of the secondary server on which llookup has to be executed.
-  String? sharedBy;
-
-  /// atSign of the secondary server for whom [atKey] is shared
-  String? sharedWith;
-
-  bool isPublic = false;
-
-  bool isCached = false;
+  late AtKey atKey;
+
```

Comment:
breaking change. Is there a way of keeping the change backwards compatible? (also the other lookup verb  builders)

Reply:
What other users of the lookup verb builders are there? Would we ever expect application code to be using these?